common/core/src/iosMain/kotlin/com/trafi/locations/LocationsApi.kt

```diff
@@ -0,0 +1,28 @@
+@file:Suppress("unused", "EXTENSION_SHADOWED_BY_MEMBER")
+
+package com.trafi.locations
+
+import com.trafi.core.ApiResult
+import com.trafi.core.CFlow
+import com.trafi.core.cFlow
+import com.trafi.core.model.AutoCompleteLocation
+import com.trafi.core.model.AutoCompleteLocations
+import com.trafi.core.model.LatLng
+import com.trafi.core.model.Location
+import com.trafi.core.model.ReverseGeocodeResponse
+
+fun LocationsApi.search(
+    query: String,
+    coordinate: LatLng? = null,
+): CFlow<ApiResult<AutoCompleteLocations>> =
+    cFlow { emit(search(query, coordinate)) }
```

Comment:
We'll generate these `fun` in near future right? 🤔

Reply:
Yes, that's the idea

Reply:
Only robots will have "fun" 😂